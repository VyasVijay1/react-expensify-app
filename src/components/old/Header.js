import React from 'react'
const Header = (props) => (
  <div>
    <h1>{props.title}</h1>
    <h1>{props.subtitle}</h1>
  </div>
)
export default Header