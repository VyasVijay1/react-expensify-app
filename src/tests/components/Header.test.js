import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/Header'
import React from 'react'
import {shallow} from 'enzyme'
import toJSON from 'enzyme-to-json'

test('should render ', () => {
 const wrapper = shallow(<Header/>)
 expect(toJSON(wrapper)).toMatchSnapshot()
 
    //const renderer = new ReactShallowRenderer();
 //renderer.render(<Header/>)
//  expect(renderer.getRenderOutput()).toMatchSnapshot()   
})