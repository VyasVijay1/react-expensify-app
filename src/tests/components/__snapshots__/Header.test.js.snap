// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render  1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <[object Object]
    activeClassName="is-active"
    to="/"
  >
    DashBoard
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/edit"
  >
    Edit Expense
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="help"
  >
    Help
  </[object Object]>
</header>
`;
